CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Added
+
+- Add support for CAPZ clusters by detecting the Azure configuration file location.
```

Comment:
Can you change this entry to make it more clear that this is only for Azure at this point?

Reply:
It specifically mentions CAPZ, isn't that clear enough?

Reply:
:joy: turns out I know nothing about provider specific Cluster API naming.